apps/ios/SimpleX Localizations/en.xcloc/Localized Contents/en.xliff

```diff
@@ -543,9 +573,72 @@
         <target>Database ID</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
-      <trans-unit id="Database export &amp; import" xml:space="preserve">
-        <source>Database export &amp; import</source>
-        <target>Database export &amp; import</target>
+      <trans-unit id="Database encrypted!" xml:space="preserve">
+        <source>Database encrypted!</source>
+        <target>Database encrypted!</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database encryption passphrase will be updated and stored in the keychain.&#10;" xml:space="preserve">
+        <source>Database encryption passphrase will be updated and stored in the keychain.
+</source>
+        <target>Database encryption passphrase will be updated and stored in the keychain.
```

Comment:
Keychain is not capital here and in some other places, not that important though..
```suggestion
        <target>Database encryption passphrase will be updated and stored in the Keychain.
```